app/assets/javascripts/components/menu_item.js.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var MenuItem = React.createClass({
+
+  navigate: function(hash) {
+    window.location.hash = hash;
+  },
+
+  render: function() {
+    var classStyle = "menu-item";
+    if (this.props.hash === "share-facebook" || this.props.hash === "get-started") {
```

Comment:
Always declare any props that a component accepts. For an example in our own code base, look [here](https://github.com/scalableinternetservices/U1F44D/blob/master/app/assets/javascripts/components/top_app_bar.js.jsx#L4); for a more complete explanation, read [this](https://facebook.github.io/react/docs/reusable-components.html).